<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_don3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>158</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="track.qrc">:/res/res/note/img_don.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_changeOrder">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>change order</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_don1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>137</width>
      <height>117</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="track.qrc">:/res/res/note/img_don.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_don2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="track.qrc">:/res/res/note/img_don.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_beat">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="track.qrc">:/res/res/track/beat_end(48x50).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="push_add">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>랜덤생성</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_del">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>랜덤삭제</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>873</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAre_you"/>
    <addaction name="actionSerious"/>
    <addaction name="actionbhbjbjb"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAre_you">
   <property name="text">
    <string>Are you</string>
   </property>
  </action>
  <action name="actionSerious">
   <property name="text">
    <string>Serious?</string>
   </property>
  </action>
  <action name="actionbhbjbjb">
   <property name="text">
    <string>bhbjbjb</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="track.qrc"/>
 </resources>
 <connections/>
</ui>
